t/app/model/user.t

```diff
@@ -30,6 +30,32 @@ is $problem->user->latest_anonymity, 0, "User's last update was not anonyous";
 create_update($problem, anonymous => 't');
 is $problem->user->latest_anonymity, 1, "User's last update was anonymous";
 
+subtest "Sign user up for alerts" => sub {
+    my $user = $problem->user;
+    my $options = {
+      cobrand      => 'default',
+      lang         => 'en-gn',
+    };
+    $user->create_alert($problem->id, %$options);
+
+    my $alert = $user->alert_for_problem( $problem->id );
```

Comment:
Just to be sure, you should check there isn't any alert at the start of this subtest (otherwise it would just be checking that create_alert doesn't create another one, which also could be something else to check).